Velog/style.css

```
@@ -0,0 +1,329 @@
+body {
+    box-sizing: border-box;
+    background-color: rgb(228, 236, 247); 
+}
+
+div {
+    display: block;
+}
+
+body {
+    margin: auto;
+    display: flex;
+    width: 100%;/*레이아웃 유연하게 지정*/
+    flex-direction: column;
+}
+
+h3 {
+    font-size: 17px;
+}
+
+h4 {
+    font-size: 14px;
+    font-weight: lighter;
+    text-align: left;
+}
+
+header {
+    display: flex;
+    align-items: center;
+    height: 100px;
+    justify-content: space-between;
+}
+
+@media all and (max-width: 500px) {
+    .card {
+        width: 80%;
+    }
+} 
+
+@media (min-width:500px) and (max-width:768px) {
+    .card {
+        width: 40%;
+    }
+}
```

```suggestion
@media all and (max-width: 600px) {
    .card {
        width: 80%;
    }
} 

@media (min-width:600px) and (max-width:768px) {
    .card {
        width: 40%;
    }
}
```

직접 테스트 해보니까 브라우저의 너비를 최대로 줄여도 `500px` 이 안됐나봐요.
`600px`로 바꾸고 해보니까 제대로 잘 동작하네요!

---

> 왁 그렇군요!! 감사합니다 ㅠㅠ 혹시 어떻게하면 브라우저 너비를 세밀하고 정확하게 지정할 수 있을까요?

미디어쿼리를 적용하는 분기점을 물어보시는걸까요? 그런거라면 [링크](http://bootstrapk.com/css/#grid-media-queries) 를 참조해보세요!

그게 아니라, 현재 내가 보고 있는 화면의 너비가 몇px 정도 될까... 하는거라면 개발자도구를 통해 알 수 있답니다!